frontend/apps/filemanager/filemanager.lua

```diff
@@ -293,8 +293,8 @@ function FileManager:init()
             {
                 {
                     text = _("Open with…"),
-                    enabled = lfs.attributes(file, "mode") == "file" and DocumentRegistry:getProviders(file) ~= nil
-                        and #(DocumentRegistry:getProviders(file)) > 1,
+                    enabled = lfs.attributes(file, "mode") == "file" and (DocumentRegistry:getProviders(file) == nil
+                        or #(DocumentRegistry:getProviders(file)) > 1),
```

Comment:
Looks like you want to remove everything after file?

Reply:
as of now it only showed  `open with` when there was 2 or more providers, i am showing it also when there are 0 providers, i did not change the behavior for 1 provider (don't show), should i do different?

Reply:
If you allow opening any file as if it were plain text it seems somewhat inconsistent to me to restrict it to unknown filetypes; not quite sure how much sense it makes one way or the other.

Reply:
to be more precise i am opening an unknown file with the provider for txt. (crengine). 
if it actually an epub for example crengine will treat it as an epub, if it is unstable crengine will probably crash.

if a file is only supported by one provider there is no reason to try opening it with a different one that probably wont work, and i only am adding this provider if there are no providers already so you will get a `open with` with the same option as opening it directly

Reply:
Right, fair enough. Carry on then. ;-)